src/Workspaces/Core/Desktop/Workspace/SQLite/SQLitePersistentStorageService.cs

```diff
@@ -98,10 +114,23 @@ private static void EnsureDirectory(string databaseFilePath)
             Directory.CreateDirectory(directory);
         }
 
+        static bool IsNativeLibraryResolutionException(Exception ex)
```

Comment:
:bulb: Missing access modifier (will send PR to add code style rule so this is more apparent)